pkgs/development/libraries/cassandra-cpp/default.nix

```diff
@@ -0,0 +1,34 @@
+{ pkgs, examples ? false}: pkgs.stdenv.mkDerivation rec {
+    pname = "cassandra-cpp";
+    version = "2.16.2";
+
+    src = pkgs.fetchgit {
+      url = "https://github.com/datastax/cpp-driver.git";
+      rev = "refs/tags/${version}";
+      sha256 = "sha256-NAvaRLhEvFjSmXcyM039wLC6IfLws2rkeRpbE5eL/rQ=";
+    };
+
+    LIBUV_ROOT_DIR = "${pkgs.libuv}/";
+    nativeBuildInputs = with pkgs; [
+      cmake
+      zlib libuv openssl.dev
```

Comment:
zlib libuv and openssl are presumably buildInputs. You can set `strictDeps = true` to check.

Reply:
Fixed.